index.html

```diff
@@ -27,9 +27,115 @@
         <a id="collaboration">
             Start collaboration
         </a>
-    </headline>
+    </h1>
 </head>
 <body>
-    
+    <h2 class="h2">
+        Projects
+    </h2>
+    <div class="grid-container">
+        <div class="flex-container">
+            <div><img id="ractangle1" src="media/Rectangle 1.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                 <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+            </ul></div>
+            <div id="stp">See this project
+                <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+           <div><img id="rectangle2" src="media/Rectangle 2.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 3.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 4.png"></div>
+            <div id="st">Project name goes here</div>
```

Comment:
- [ ] please The project title (e.g., project name goes here) use a header tag hx(2,3,4)